static/js/components/Group.jsx

```diff
@@ -263,7 +263,7 @@ const Group = () => {
                     <Chip label="Admin" size="small" color="secondary" />
                   </div>
                 )}
-                {isAdmin(currentUser) &&
+                {currentUser.acls?.includes("Manage users") &&
```

Comment:
The user should still be an admin of the group, so please add `isAdmin(currentUser) &&` back. I guess I wasn't clear when filing that issue, sorry about that.